app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -184,7 +184,7 @@ examples:
         text: "Blue"
   with_custom_heading_size:
     description: |
-      This allows the size of the legend to be changed. Valid options are s, m, l, xl, defaulting to m if no option is passed. 
+      This allows the size of the legend to be changed. Valid options are s, m, l, xl, defaulting to m if no option is passed.
 
       If the is_page_heading option is true and heading_size is not set, the text size will be xl.
```

Comment:
Since we've now got this option, the `is_page_heading` option is a bit redundant/confusing. I think it would be better if these options were consolidated together, so if a heading size is passed, it gets rendered as that heading, otherwise it's not a heading. Then there'd be one option for setting the heading level, and one option for setting the heading size.

This is outside the scope of this PR but maybe worth raising an issue about?

Reply:
Totally agreed, have raised an issue to further consolidate the component https://github.com/alphagov/govuk_publishing_components/issues/1953